packages/contracts/README.md

```diff
@@ -16,35 +16,27 @@ yarn add @aragon/osx-artifacts
 
 To get started running your repository locally:
 
-Copy `.env.example` into a file called `.env` or create a new one with these 3 keys defined:
+1. Install pacakages from the root folder with `yarn`
```

Comment:
```suggestion
1. Install packages from the root folder with `yarn`
```

Reply:
to avoid typo in the future please install a spell checker extension such as: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker